pkg/statistics/handle/storage/gc.go

```diff
@@ -131,35 +131,41 @@ func GCStats(
 
 // DeleteTableStatsFromKV deletes table statistics from kv.
 // A statsID refers to statistic of a table or a partition.
-func DeleteTableStatsFromKV(sctx sessionctx.Context, statsIDs []int64) (err error) {
+func DeleteTableStatsFromKV(sctx sessionctx.Context, statsIDs []int64, lowPriority bool) (err error) {
 	startTS, err := util.GetStartTS(sctx)
 	if err != nil {
 		return errors.Trace(err)
 	}
+	updataStmt := "update "
+	deleteStmt := "delete "
+	if lowPriority {
```

Comment:
Better to add some comments to explain the reason.

Reply:
I remember these priority options have no effect in fact 😂 

https://github.com/pingcap/tidb/issues/51104